docs/advanced-concepts/route-narrowing.md

```diff
@@ -19,29 +19,29 @@ const user = createRoute({
 
 const profile = createRoute({
   parent: user,
-  name: 'profile',
+  name: 'user.profile',
   path: '/profile',
   query: '?tab=[tab]',
   component: ...,
 })
 
 const settings = createRoute({
   parent: user,
-  name: 'settings',
+  name: 'user.settings',
   path: '/settings',
   component: ...,
 })
 
 const router = createRouter([home, user, profile, settings])
 
-router.route.key // "home" | "user" | "user.profile" | "user.settings"
+router.route.name // "home" | "user" | "profile" | "settings"
```

Comment:
I think this contradicts the change above where you made the names dot notation user.settings and user.profile

Reply:
oops, your right